akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md

```diff
@@ -52,11 +51,11 @@ Java
 
 **emits** when the futures (in submission order) created by the ask pattern internally are completed
 
-**backpressures** when the number of futures reaches the configured parallelism and the downstream backpressures
+**backpressure** when the number of futures reaches the configured parallelism and the downstream backpressure
```

Comment:
'backpressure' is intended to be a verb here, so it needs to be 'backpressures' in both cases.